api/settings.md

```diff
@@ -16,6 +16,8 @@ Only requesting these settings is possible. Changing them is not.
 ```sh
 $ curl {{ site.parku.api }}/settings \
     -u 098f6bcd4621d373cade4e832627b4f6:parku \
+    -d lat=52.53536625 \
+    -d lon=13.39387752 \
```

Comment:
Can you add that to the query string as seen on the [location endpoint](http://developer.parku.ch/api/locations/#http-request) and not use the curl parameter `-d`.